reviewing/dependencies-hierarchy/test/getTree.test.ts

```diff
@@ -401,29 +401,31 @@ describe('getTree', () => {
     test('height > requestedDepth', () => {
```

Comment:
Looking over this again, it looks like this never tested the fully visited cache. The `a` package was never added to the fully visited cache since the dependencies of its leaf nodes were never enumerated.

Adding a few comments to make this more clear and increasing the max depth of the test case.

The other test cases were okay.